.github/workflows/msys2_ffmpeg.yml

```diff
@@ -0,0 +1,126 @@
+name: MSYS2 FFmpeg Plugins
+on: [push, pull_request, workflow_call]
+      
+env:
+  FFMPEG_PREFIX: /c/ffmpeg_mtl
+  
+jobs:
+  ffmpeg_build:
+    name: MSYS2 FFmpeg Build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sys:
+          - mingw64
+        dpdk: [23.03]
+    defaults:
+      run:
+        shell: msys2 {0}        
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      
+      - id: ffmpeg_compose
+        uses: OpenVisualCloud/Media-Transport-Library@main
```

Comment:
Is this means the action will use the main code, not the commits within the PR?

Reply:
This line here indicates that is calling the composite action resides on main branch 
https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/action.yml

Reply:
ok, then it's wrong. The workflow for PRs should be run against the PR commits.

Reply:
Let me rephrase, the workflow still checkout the latest PR commits for IMTL with
uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2

The usage of this liner is basically lookup for action.yml to call another workflow in the existing workflow
uses: OpenVisualCloud/Media-Transport-Library@main

Reply:
Ok, thanks to clarify this.